app/views/teams/show.html.haml

```diff
@@ -6,15 +6,15 @@
 
 %table.table.table-condensed.table-striped
   %thead
-  %tr
-    %th.span1= t('teams.team_users_table.team_leader_header')
-    %th= t('teams.team_users_table.email_header')
-    %th= t('teams.team_users_table.grade')
-    %th.span2= t('teams.team_users_table.prize_eligibility_header')
-    %th= t('teams.team_users_table.remove_user_header')
-    - if @team_captain
-      %th
-        = t('teams.team_users_table.change_captian_header')
+    %tr
+      %th.span1= t('teams.team_users_table.team_leader_header')
```

Comment:
It seems a little reduntant here for the parent key to be `teams` and then the child key to start with `team`, maybe rename this to `teams.user_table.team_leader_header`?